draft-mvp/src/App.js

```diff
@@ -20,12 +21,22 @@ class App extends Component {
     this.state = {
       nbaTeams: NBATEAMS,
       college: COLLEGE,
-      players: PLAYERS
+      players: PLAYERS,
+      selectedNbaTeams: []
     };
+    this.handleSelectNbaTeams = this.handleSelectNbaTeams.bind(this);
   }
 
+  //update which nba team to render on page
+  handleSelectNbaTeams = selectedNbaTeam => {
+    this.setState({
+      selectedNbaTeams: this.state.selectedNbaTeams.concat([selectedNbaTeam])
+    });
+  };
+
```

Comment:
Since you are using ES6 Arrow functions there is no need to bind the value of "this" again on line 27.